client/src/App.js

```diff
@@ -16,7 +23,28 @@ function App() {
 
   return (
     <div className="App">
-      <h1> {text}</h1>
+      <header className="App__header">
+        <Navigation />
+      </header>
+      <main className="App__content">
+        <Switch>
+          <Route path="/listCakes/:id">SingleCakePage</Route>
+          <Route path="/menu">
+            <Menu />
+          </Route>
+          <Route path="/add">
+            <CreateNewPage />
+          </Route>
+
+          <Route path="/">
+            <Home />
+          </Route>
+        </Switch>
+      </main>
+
+      <footer classname="App__footer">
+        <Footer />
+      </footer>
```

Comment:
Hi Boris, instead of wrapping the Footer component in footer tags, you could have also put the `<footer>` tags inside the component. when you do this you can remove the div with classname "footer" inside the component.

Reply:
Hi John, you´re right. Somehow I missed this when I was coding.